.env.example

```diff
@@ -1,30 +1,30 @@
-export DEPLOYMENT_TYPE="development"    # Deployment type (development, staging, production)
+DEPLOYMENT_TYPE="development"    # Deployment type (development, staging, production)
 
-export KALEMAT_API_KEY=""          # Token for Qur'an and Hadith search
-export ANTHROPIC_API_KEY=""        # API key for Claude AI model
-export OPENAI_API_KEY=""           # Token for GPT-4
+KALEMAT_API_KEY=""          # Token for Qur'an and Hadith search
+ANTHROPIC_API_KEY=""        # API key for Claude AI model
+OPENAI_API_KEY=""           # Token for GPT-4
 
 # Optional. If not set, the app will not use these services.
-export SENDGRID_API_KEY=""         # API key to send password reset options
+SENDGRID_API_KEY=""         # API key to send password reset options
 
 # Database connection string
-export MONGO_URL="mongodb://localhost:27017"
-export MONGO_DB_NAME="ansari_db"
+MONGO_URL="mongodb://localhost:27017"
+MONGO_DB_NAME="ansari_db"
 
-export SECRET_KEY="secret"         # Secret key for signing tokens
+SECRET_KEY="secret"         # Secret key for signing tokens
 
 # Origins to be allowed by the backend
-export ORIGINS="https://beta.ansari.chat,http://beta.ansari.chat,https://ansari.chat,http://ansari.chat,https://hajiansari.ai,http://hajiansari.ai,https://ansari.endeavorpal.com,https://web.whatsapp.com,https://localhost,capacitor://localhost"
+ORIGINS="https://beta.ansari.chat,http://beta.ansari.chat,https://ansari.chat,http://ansari.chat,https://hajiansari.ai,http://hajiansari.ai,https://ansari.endeavorpal.com,https://web.whatsapp.com,https://localhost,capacitor://localhost"
```

Comment:
Should we clean this up? At least for this demo, can we just include https://ansari.chat, http://localhost etc?